.travis.yml

```diff
@@ -10,15 +10,5 @@ env:
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
 addons:
   postgresql: '9.4'
-before_install:
-- source ${TRAVIS_BUILD_DIR}/tools/ci/before_install.sh
-script:
-- bundle exec bin/rails app:test:providers:lenovo:setup app:test:providers:lenovo
-after_script: bundle exec codeclimate-test-reporter
-notifications:
-  webhooks:
-    urls:
-    - https://webhooks.gitter.im/e/994cc93494715ae5990f
-    on_success: change
-    on_failure: always
-    on_start: never
+install: bin/setup
+after_script: bundle exec codeclimate-test-reporter
```

Comment:
Ah yeah, I think @durandom mentioned that it's missing in the generator, so I'll add it there too.